Simplified/StringUtilities.m

```diff
@@ -0,0 +1,26 @@
+//
+//  StringUtilities.m
+//  Simplified
+//
+//  Created by Aferdita Muriqi on 6/20/17.
+//  Copyright © 2017 NYPL Labs. All rights reserved.
+//
+
+#import "StringUtilities.h"
+
+@implementation StringUtils
+
+NSString * NYPLLocalizedString(NSString * key, NSString * comment) {
```

Comment:
Despite my other comment, it seems we do not actually need a class here at all! The only thing contained in `StringUtils` is a top-level function, not a method, and thus we should probably just define the function at the top level and dispatch with the class altogether.